Launcher/Classes/ViewControllers/InspectorViewController.swift

```diff
@@ -352,7 +352,7 @@ extension InspectorViewController: NSOutlineViewDataSource {
             return uiElements.count
         }
 
-        let currentChildIndex = uiElements.index(of: item as! String)!
+        guard let currentChildIndex = uiElements.index(of: item as! String) else { return 0 }
```

Comment:
We should also try to get rid of the `as!` cast here, as that would mean if the `item` is not a `String`, it would still crash.